src/www/services_dhcp.php

```diff
@@ -985,7 +985,7 @@ function show_netboot_config() {
 											</div>
 											<div id="showldap" style="display:none">
 												<input name="ldap" type="text" class="form-control unknown" id="ldap" size="80" value="<?=htmlspecialchars($pconfig['ldap']);?>" /><br />
-												<?=gettext("Leave blank to disable.  Enter a full URI for the LDAP server in the form ldap://ldap.example.com/dc=example,dc=com");?>
+												<?=gettext("Leave blank to disable. Enter a full URI for the LDAP server in the form ldap://ldap.example.com/dc=example,dc=com");?>
```

Comment:
Can we have the sample URI as %s , makes it easier not to screw up the translation by accident.